vue/configurator/src/views/RestrictedTokenView.vue

```diff
@@ -33,6 +33,37 @@ export default class IdentityView extends Vue {
                 value: "", // can test with 0xf11f75c3b4548138c8894df6f32517c82fa7e362
                 required: true,
               },
+              /* Schemed Utility class definitons for light/dark
+              https://docs.manifold.xyz/v/manifold-for-developers/resources/widgets/manifold-css-variables/scheme-utility-classes */
+              "data-widget-theme": {
+                name: "Widget Color",
+                type: WidgetPropType.ENUMERATION,
+                value: "",
+                options: [
+                  {
+                    value: "manifold-scheme-dark",
+                    label: "Dark",
+                  },
+                  {
+                    value: "manifold-scheme-light",
+                    label: "Light",
+                  },
+                  // prefers-color-scheme: queries media based on the light/dark mode in browser
+                  // Tested using Microsoft Edge and Chrome
+                  // https://developer.mozilla.org/en-US/docs/Web/API/Window/matchMedia
+                  {
+                    value: (
+                      window.matchMedia &&
+                      window.matchMedia("(prefers-color-scheme: dark)")
+                    ).matches
+                      ? "manifold-scheme-dark"
+                      : "manifold-scheme-light",
+                    label: "OS Preference",
+                  },
```

Comment:
this value is unnecessary, based on the [scheme utility classes docs](https://docs.manifold.xyz/v/manifold-for-developers/resources/widgets/manifold-css-variables/scheme-utility-classes), since by default, the widgets will render according to the user's os pref. you can just set value = '' and test for that on the configurator (don't add a class value if the theme value is empty).

Reply:
Ahh I see, I think I misinterpreted the OS preference as an active state being pulled when the user chooses to set it to their preference. Instead it's more the default value the configurator loads as? And thus, the `value=""` ; label would be OS Preference

Reply:
Updated!